lib/ecto/repo.ex

```diff
@@ -1760,7 +1760,7 @@ defmodule Ecto.Repo do
         repo.insert!(%Post{})
       end)
 
-  If an Elixir exception occurs the transaction will be rolled back
+  If an exception occurs the transaction will be rolled back
```

Comment:
Databases also have exceptions. So since you said errors were confusing, it doesn't hurt to be double explicit. :)

```suggestion
  If an Elixir exception occurs the transaction will be rolled back
```

Reply:
Isn’t the transaction rolled back in both cases here? So shouldn’t we understand “exceptions” as both application exceptions and database exceptions?

The original text was about unhandled erros and I think that this gave a more generic understanding that it could be from application or database. WDYT?

Reply:
The rest of the sentence does not apply to DB transactions.